packages/uniforms-material/src/DateField.tsx

```diff
@@ -40,11 +39,11 @@ function Date({
     <TextField
       disabled={disabled}
       error={!!error}
-      fullWidth={fullWidth}
+      fullWidth
       helperText={(error && showInlineError && errorMessage) || helperText}
       label={label}
       InputLabelProps={{ ...labelProps, ...InputLabelProps }}
-      margin={props.margin ?? 'dense'}
+      margin="dense"
```

Comment:
I guess we usually use jsx string props in curly brackets like `{"dense"}`

Reply:
We’ve agreed it’s correct and created [a card](https://github.com/orgs/vazco/projects/1#card-50180301) to configure it in our ESLint config.